### PR TITLE
Add Apache Sedona example

### PR DESCRIPTION
**Pull request by jiayuasu**
Dear all,

Thanks for the release of the OMF data. This is really helpful to our community. I really appreciate it!

This PR is to add the example of reading OMF data via Apache Sedona (https://sedona.apache.org/). Sedona provides both Python and Spatial SQL API. I believe this will be a good addition to this repository.

Thanks,
Jia

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,8 +185,27 @@ SET s3_region='us-west-2';
...
 
-### 4. Download the Parquet files
+You can get a single-node Sedona Docker image from [Apache Software Foundation DockerHub](https://hub.docker.com/r/apache/sedona) and run `docker run -p 8888:8888 apache/sedona:latest`. A Jupyter Lab and notebook examples will be available at http://localhost:8888/. You can also install Sedona to Databricks, AWS EMR and Snowflake using [Wherobots](https://www.wherobots.ai/demo).
+
+The following Python + Spatial SQL code reads the Places dataset and runs a spatial filter query on it.
+
+```
+import sedona.spark.*
+
+config = SedonaContext.builder().getOrCreate()
```
Comment (jenningsanderson) [inline README.md:197]: In order to access the files, I had to use this: 

```
config = SedonaContext.builder().config("fs.s3a.aws.credentials.provider", 
                                        "org.apache.hadoop.fs.s3a.AnonymousAWSCredentialsProvider").getOrCreate()
sedona = SedonaContext.create(config)
```

File: README.md [hunk 2]
```diff
@@ -185,8 +185,27 @@ SET s3_region='us-west-2';
...
 
+### 4. Apache Sedona (Python + Spatial SQL)
 
-### 4. Download the Parquet files
+You can get a single-node Sedona Docker image from [Apache Software Foundation DockerHub](https://hub.docker.com/r/apache/sedona) and run `docker run -p 8888:8888 apache/sedona:latest`. A Jupyter Lab and notebook examples will be available at http://localhost:8888/. You can also install Sedona to Databricks, AWS EMR and Snowflake using [Wherobots](https://www.wherobots.ai/demo).
+
+The following Python + Spatial SQL code reads the Places dataset and runs a spatial filter query on it.
+
+```
+import sedona.spark.*
```
Comment (jenningsanderson) [inline README.md:195]: `from sedona.spark import *`

**Review by jenningsanderson — CHANGES_REQUESTED** [review]
I had to make these changes to run it on my machine, but once I made them, it worked great!

File: README.md (hunk 2 above)
Comment (jiayuasu) [reply README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (jiayuasu) [reply README.md]: Fixed.

**Comment by jiayuasu** [comment]
@jenningsanderson Thanks for your comments. Please review again :-)

**Review by jenningsanderson — APPROVED** [review]
Works for me, very cool!

**Comment by vcschapp** [comment]
Late to the party, just stopped by to say thanks @jiayuasu (and @jenningsanderson for the review).

**Pull request merged**